### ISSUE TITLE
When throwing windows between spaces in floating layout, retain size

### ISSUE THREAD

**Issue by ericauld** [ianyh/Amethyst#1336]
When I throw windows between spaces, and both spaces have a floating layout, the thrown window then fills up the screen. It'd be great if it kept its size and placement on the screen. 

I'm running Ventura 13.0 on a 2020 MacBook Pro with M1. I also have Better Touch Tool running, which has been known to make mischief from time to time. I have turned off the option "Automatically rearrange spaces based on most recent use".

**Comment by jli** [comment]
I have also noticed an issue where I configure all windows to be floating, but the "Throw window to space N" action moves the window and also maximizes the window to fill the desktop. I also noticed that disabling "Follow thrown windows between spaces" doesn't work: i have that setting disabled, but when I use the "throw window to space N" actions, I get switched to the space I threw the window to (whereas I expected to stay in the same space, and just have the window move).

I see this with Amethyst 0.16.1 on an M1 Macbook Pro 2021.
I downgraded Amethyst to 0.15.6 and did not experience this bug.
